.github/workflows/publish-nightly.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish (nightly)
+
+on:
+  schedule:
+    # Every day 00:00 UTC
+    - cron:  '0 0 * * *'
+
+jobs:
+  publish:
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: 📥 Checkout repository
+      uses: actions/checkout@v2.3.3
+      with:
+        # Fetch all history for all tags and branches
+        fetch-depth: 0
+        ref: nightly
+        submodules: true
+      run: git rebase master
+
+    - name: 🧰 Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.17
+        registry-url: 'https://registry.npmjs.org'
+
+    # Alternatively (if we don't like the workflow failures)
```

Comment:
Let's go with the alternative, as a failure indicates something is wrong and needs attention.

Reply:
Done in 8fbf2d3d81f571ece14746db9f0a77d2f7fd6751